webservices/partition/base.py

```diff
@@ -203,8 +195,7 @@ def create_constraints(cls, child, cycle, temp=True):
             'alter table {child} alter column {primary} set not null',
             'alter table {child} add primary key ({primary})',
             'alter table {child} alter column filing_form set not null',
-            'alter table {child} add constraint check_two_year_transaction_period check (two_year_transaction_period in ({start}, {stop}))',  # noqa
-            'alter table {child} inherit {master}'
+            'alter table {child} add constraint check_two_year_transaction_period check (two_year_transaction_period in ({start}, {stop}))'  # noqa
```

Comment:
The constraint shouldn't need to be defined here as it should already been on the child table given the SQL changes made above.  Also, see note above about removing this entire method (`create_child`).